articles/tech-resources/inbound-email-relay-webhook.md

```diff
@@ -35,6 +35,19 @@ Once you have these, there are three steps to setting up your Relay Webhook:
 * setting up an Inbound Domain in SparkPost
 * setting up a Relay Webhook in SparkPost
 
+**Note**: You *cannot* use a domain already in use as a bounce domain as an inbound domain. An attempt to create an inbound domain via the API when this condition is present will yield a `409 conflict error` as a result:
```

Comment:
Should we put "already verified as a bounce domain" versus "already in use as a bounce domain".  You can have it created, but not verified as a bounce domain, and insert it as an inbound domain without error.